Tests/SwiftStdlibTests/RawRepresentableExtensionsTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  RawRepresentableExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by 方林威 on 2020/7/1.
+//  Copyright © 2020 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+private enum Animal: String {
```

Comment:
I think you can move this inside the `testOptionalRawValueInitializer` function.

Reply:
Yes, it can indeed be in the testOptionalRawValueInitializer function